test/extensions/filters/common/ext_authz/ext_authz_impl_test.cc

```diff
@@ -108,7 +110,7 @@ TEST_F(ExtAuthzGrpcClientTest, BasicError) {
 
   client_.check(request_callbacks_, request, Tracing::NullSpan::instance());
 
-  EXPECT_CALL(request_callbacks_, onComplete(CheckStatus::Error));
+  EXPECT_CALL(request_callbacks_, onComplete_(_));
```

Comment:
we should be checking the Response's `CheckStatus` in this file.

Reply:
Definitely, I should have put a comment explaining that it's temporary. Thanks!